### PR TITLE
Mark 'is origin trustworth' as exported definition of an algorithm

### PR DESCRIPTION
**Pull request by dontcallmedom**
used in https://w3c.github.io/network-error-logging/

### CODE REVIEW

**Review by mikewest — APPROVED** [review]
Yes, this should be exported. Thanks for catching it.

File: index.bs [hunk 1]
```diff
@@ -525,7 +525,7 @@ urlPrefix: https://www.w3.org/2017/Process-20170301/; spec: W3C-PROCESS
 <section>
   <h2 id="algorithms">Algorithms</h2>
 
-  <h3 id="is-origin-trustworthy">
+  <h3 dfn-type="algorithm" export id="is-origin-trustworthy">
```
Comment (mikewest) [inline index.bs:528]: Bikeshed doesn't like the `dfn-type="algorithm"` annotation.

Comment (dontcallmedom) [reply index.bs:528]: sorry, I always get confused - the type is "abstract-op" not algorithm; not sure why my local bikeshed didn't complain though

**Comment by mikewest** [comment]
Thanks!

**Comment by carlaarchuleta707-lgtm** [comment]
https://patch-diff.githubusercontent.com/raw/w3c/webappsec-secure-contexts/pull/102.diff#:~:text=diff%20%2D%2Dgit%20a,potentially%20trustworthy%3F%0A%20%20%20%3C/h3%3E

**Comment by carlaarchuleta707-lgtm** [comment]
turbo.es2017-esm.js

**Pull request merged**